src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingBaker.java

```diff
@@ -14,6 +15,9 @@
 import com.minecolonies.coremod.entity.ai.citizen.baker.BakerRecipes;
 import com.minecolonies.coremod.entity.ai.citizen.baker.BakingProduct;
 import com.minecolonies.coremod.entity.ai.citizen.baker.ProductState;
+import com.minecolonies.coremod.network.messages.AssignBakerRecipeMessage;
+
+import io.netty.buffer.ByteBuf;
 import net.minecraft.block.Block;
```

Comment:
### Errors:
- Unused import import net.minecraft.block.Block;

Reply:
These import are required and will not compile without them.